### ISSUE TITLE
PM | Editing as a subtask | KPI / analytics check

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#63550]
## Background 

As we prepare for launch we need to have our metrics/GA/Domo dashboard updated for the new feature.

## Tasks 
- [x] [discuss kpi's w/PO](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/mobile-phone-edit-flow/README.md#measuring-success)
- [x] [create the analytics review ticket for the collab cycle ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63770)

**Comment by mtcA6** [comment]
@Samara-Strauss thoughts on the [KPIs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/contact-information/mobile-phone-edit-flow/README.md#measuring-success) I put in the brief?

**Comment by Samara-Strauss** [comment]
I'll need to think more about this. The goal of this work isn't necessarily to increase the number of mobile numbers we have on file, and it may or may not make a difference with regards to that. I'm also unsure about what you mean by "percentage of users using the legacy feature." 

A couple thoughts:

1. It would be great to know if we increase task completion for adding a mobile number from Notification Preferences by switching to the new flow. I believe we'd need a funnel report for this, which we don't have this set up now, but I guess we could talk to analytics about this.
2. We should also establish baseline metrics for how many successful saves there are of this form.

**Comment by mtcA6** [comment]
@Samara-Strauss No worries, noodle away!!! 

I should have given you a bit more detail about my thoughts and I'm sorry for the imprecision of my language. When I said legacy feature, to me, that means users updating via the Contact Information screen vs. the subtask flow. 

As I understood things, the editing as a subtask feature goal was to reduce the cognitive load for users since it was a bit jarring the way it jumps a user between screens, but I can't think of a way to measure that. 

So, my thoughts were:
- we can see if this increases the number of profiles with a mobile phone number (Percentage of users missing mobile phone number, was hoping to use that data Mike R. shared), I would hope this number would go down, but I'm not confident it will, but I'm interested to track this
- we can also see if this reduces user's need to navigate to the Contact Info page (reduced traffic to contact information page, thinking we could use a ratio based on the number of unique visitors and how many of those use the contact info screen prior to release) 
- we can track the number of users who use the subtask flow (Percentage of users updating using the new feature [successful subtask updates/all mobile phone number updates]).

**Comment by Samara-Strauss** [comment]
TL;DR -- whatever metrics we look at, we should be fairly sure that we're seeing the effects of our work as metrics change. If we can't be, the metrics we look at are meaningless. After doing some thinking, I think we can look at the following:

1. Reduced ranking for Notification preferences as an entrance page to Contact info. ([Report](https://analytics.google.com/analytics/web/?authuser=0#/report/content-pages/a50123418w177519031p176188361/explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.pagePath:www.va.gov~2Fprofile~2Fcontact-information~2F&_r.tabId=navigationsummary/))
2. Baseline metrics for how many people complete the new subtask flow from Notification prefs. 

-------

> we can see if this increases the number of profiles with a mobile phone number (Percentage of users missing mobile phone number, was hoping to use that data Mike R. shared), I would hope this number would go down, but I'm not confident it will, but I'm interested to track this

Mobile phone updates aren't going to tell us much of anything. This flow is not designed to increase phone updates at scale. Additionally, someone could add a mobile phone number by other means than VA.gov. VA Profile connects to many systems at VA, so if someone adds a mobile phone number or updates their address elsewhere, that will propagate to VA.gov next time someone logs in. In other words, too many factors affect the mobile phone totals for us to be able to extrapolate whether this new subtask flow had a major effect or not.

> we can also see if this reduces user's need to navigate to the Contact Info page (reduced traffic to contact information page, thinking we could use a ratio based on the number of unique visitors and how many of those use the contact info screen prior to release)

I also don't know that reduced traffic to the Contact info page as a whole will tell us much of anything, since traffic to the Contact info page also can vary month to month with no changes to the UI. 

One thing we could look at is how much of the entrance traffic to Contact info is coming from Notification preferences. [In this report, you can see that the Notification preferences page is the 4th-most entrance page for Contact info for the last month](https://analytics.google.com/analytics/web/?authuser=0#/report/content-pages/a50123418w177519031p176188361/_u.date00=20230714&_u.date01=20230813&explorer-table.plotKeys=%5B%5D&_r.drilldown=analytics.pagePath:www.va.gov~2Fprofile~2Fcontact-information~2F&_r.tabId=navigationsummary/). With editing as a subtask, I'd expect to see that ranking drop significantly (maybe not entirely, because someone might go to Contact info after Notification preferences for an entirely different reason than adding a mobile phone number). I think this could be a reliable KPI.

> we can track the number of users who use the subtask flow (Percentage of users updating using the new feature [successful subtask updates/all mobile phone number updates]).

I think we should establish baseline metrics of how many people use the subtask flow, but we don't need to do any fancy math with it. Again, I don't think this number compared to total mobile phone numbers will tell us much of anything.

**Comment by mtcA6** [comment]
@Samara-Strauss updated the brief. I'll submit the analytics ticket here in short order. 

Thanks for the help!
